packages/features/bookings/lib/handleNewBooking/test/fresh-booking.test.ts

```diff
@@ -140,7 +141,6 @@ describe("handleNewBooking", () => {
         const calendarMock = mockCalendarToHaveNoBusySlots("googlecalendar", {
           create: {
             id: "MOCKED_GOOGLE_CALENDAR_EVENT_ID",
-            iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
```

Comment:
We shouldn't remove this. By removing this, AFAIU, we are mocking a scenario where iCalUID isn't returned by Google Calendar which it normally does.

It could be a different test where Google Calendar doesn't return iCalUID, so we could copy this test.

Reply:
When passing a iCalUID to Google Calendar, we already know what it is. I've made some changes to the code where it won't return the iCalUID from Google Calendar because we were the ones that set it.

Reply:
I don't see those changes in this PR. But quickly checked and I guess they are part of the target branch in PR https://github.com/calcom/cal.com/pull/12122/files#diff-6bfbaf63c841dc9343dba0c7487e30a546ffe2a63c9d9418379d0dbb88644d27.
So, it should be fine then.